charts/kube-master/templates/_openstack.config.tpl

```diff
@@ -13,6 +13,7 @@ region = {{ .Values.openstack.region }}
 [LoadBalancer]
 lb-version=v2
 subnet-id= {{ required "missing openstack.lbSubnetID" .Values.openstack.lbSubnetID }}
+floating-network-id= {{ required "missing openstack.lbFloatingNetworkID" .Values.openstack.lbFloatingNetworkID }}
```

Comment:
for 1.7 this is not strictly *required*. But maybe we collect it any case.

Reply:
I didn't want to litter too many `if`s throughout the code. De facto `1.7` won't be used anymore anyway...